File: src/main/java/de/meshcloud/example/springkotlin/repositories/ResourceRepository.java
package de.meshcloud.example.springkotlin.repositories;

import de.meshcloud.example.springkotlin.model.Resource;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.rest.core.annotation.RepositoryRestResource;

@RepositoryRestResource
public interface ResourceRepository extends JpaRepository<Resource, Long> {
}

File: src/main/kotlin/de/meshcloud/example/springkotlin/repositories/ResourceRepository.kt
package de.meshcloud.example.springkotlin.repositories

import de.meshcloud.example.springkotlin.model.Resource
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.rest.core.annotation.RepositoryRestResource

@RepositoryRestResource
interface ResourceRepository : JpaRepository<Resource, Long>
